R/legend.R

```diff
@@ -99,8 +99,9 @@ draw_legend <- function(gsplot) {
         #set bg so that fill/border/etc args are correct, then evaluate any quoted list items
         if (any(names(legend) %in% c("bg"))) {
           par(bg=legend$bg)
+        } else {
+          par(bg = ifelse(par('bg') == "transparent", "#FFFFFF", par('bg')))
         }
-        legend <- lapply(legend, function(x) {unname(sapply(x, function(x) {eval(x)}))})
```

Comment:
agreed, I think that's a pasting error on my part, I certainly never planned to delete that line.